packages/p2p-v2/src/components/RadioGroup/RadioGroup.tsx

```diff
@@ -4,23 +4,23 @@ import { Text } from '@deriv-com/ui/dist/components/Text';
 import './RadioGroup.scss';
 
 type TItem = HTMLAttributes<HTMLDivElement> & {
-    id?: string;
```

Comment:
Any reason behind updating this file from P2P package?

Reply:
This has come after pulling the latest changes of master, let me fix it